src/engine/Engine.ts

```diff
@@ -232,8 +232,8 @@ export class Engine extends Class {
     * Default [[IEngineOptions]]
     */
    private static _DefaultEngineOptions: IEngineOptions = {
-      width: 0,
-      height: 0,
+      canvasWidth: 0,
```

Comment:
I agree we should avoid changing the IEngineOption. 

I wouldn't recommend `getScreenWidth()`, as that could potentially be mistaken for something else (e.g. the width of the browser window or actual screen dimensions). What about`getCanvasWidth()` for "the width of the game canvas in pixels" and `getPixelWidth()` for "the width of the engine's drawing surface"?